roles/ansible/tower/config-ansible-tower-subscription-manifest/tasks/subscription-manifest.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: "Wait for Tower to become available before proceeding (30 sec max)"
+  uri:
+    url: "{{ ansible_tower.url | default(default_ansible_tower_url) }}"
+    user: "{{ ansible_tower.admin_username | default(default_ansible_tower_admin_username) }}"
+    password: "{{ ansible_tower.admin_password }}"
+    force_basic_auth: yes
+    method: GET
+    validate_certs: "{{ ansible_tower.validate_certs | default(validate_tower_certs) | default(true) }}"
+  register: status_output
+  until: status_output.status == 200
+  retries: 6
```

Comment:
Please make the retries/delay configurable so we can easily update in inventories if/when needed + we may want to increase the default to more than 30 sec .... when deploying to OCP I noticed it takes more than 30 sec before it's ready due to the installation being in an "upgrade/install" mode for awhile.